features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -0,0 +1,81 @@
+Feature: cluster monitoring related upgrade check
+  # @author hongyli@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: upgrade cluster monitoring along with OCP - prepare
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-monitoring" project
+    When I run the :apply client command with:
+      | f          | <%= BushSlicer::HOME %>/features/upgrade/monitoring/cm-monitoring-retention.yaml |
+      | overwrite  | true |
+    Then the step should succeed
+
+  # @author hongyli@redhat.com
+  # @case_id OCP-29797
+  @upgrade-check
+  @admin
+  Scenario: upgrade cluster monitoring along with OCP
+    Given I switch to cluster admin pseudo user
+    And I use the "openshift-monitoring" project
+    When I run the :get client command with:
+      | resource | pods |
+    Then evaluation of `@result[:stdout].split(/\n/).drop(1).map {|n| n.split(/\s+/)[2]}` is stored in the :pod_status clipboard
+    And I repeat the following steps for each :status in cb.pod_status:
+    """
+    Then the expression should be true> cb.status == "Running"
+    """
+    When I run the :get client command with:
+      | resource | clusteroperator/monitoring |
```

Comment:
It's better change to
| resource              | clusteroperator |
| resource_name   | monitoring         |